weather_app/weather/templates/weather/vistapost.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html>
+<head>
+<style>
+ .button {
+      background-color: #008CBA;
+      border: none;
+      color: white;
+      padding: 15px 32px;
+      text-align: center;
+      text-decoration: none;
+      display: inline-block;
+      font-size: 16px;
+      margin: 4px 2px;
+      cursor: pointer;
+    }    
+    
+</style>
+</head>
+<body>
+
+<h1>Formulario De Registro</h1>
+
+<form action="" method="post">
+    {% csrf_token %} 
+  <label for="primernombre">Primer Nombre:</label>
+  <input type="text" id="primernombre" name="primer_nombre"><br><br>
+  <label for="segundonombre">Segundo Nombre:</label>
+  <input type="text" id="segundonombre" name="segundo_nombre"><br><br>
+  <label for="primerapellido">Primer Apellido:</label>
+  <input type="text" id="primerapellido" name="primer_apellido"><br><br>
+  <label for="segundoapellido">Segundo Apellido:</label>
+  <input type="text" id="segundoapellido" name="segundo_apellido"><br><br>
+
+  <p>Seleciona El Tipo De Documento:</p>
+  <input type="radio" id="registro" name="tipo_documento" value="R">
+  <label for="registro">Registro Civil De Nacimiento</label><br>
+  <input type="radio" id="tarjeta" name="tipo_documento" value="TI">
+  <label for="tarjeta">Tarjeta De Indentidad</label><br>
+  <input type="radio" id="cedula" name="tipo_documento" value="CC">
+  <label for="cedula">Cedula</label><br>
+  <input type="radio" id="tarjetae" name="tipo_documento" value="TE">
+  <label for="tarjetae">Tarjeta De Extranjeria</label><br>
+  <input type="radio" id="cedulae" name="tipo_documento" value="CE">
+  <label for="cedulae">Cedula De Extranjeria</label><br>
+  <input type="radio" id="pasaporte" name="tipo_documento" value="PA">
+  <label for="pasaporte">Pasaporte</label><br><br>
+.
+  <label for="numeroIdentidad">Numero de identidad</label>
+  <input type="number" id="numeroIdentidad" name="numeroIdentidad"><br><br>
```

Comment:
```suggestion
  <input type="number" id="numero_identidad" name="numero_identidad"><br><br>
```

Usualmente para los nombres y IDs se utilizan los valores delimitados por (_) todo te funciona ok pero para que lo miremos en los proximos IDs y names